src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -64,7 +64,7 @@ private SyntaxNode GetFixedNode(dynamic fixedRoot)
             await TestMissingAsync(@"class C { int [||]x = 42; void M() { System.Console.WriteLine(x); } }");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/12838"), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
         public async Task WithRefInitializer1()
         {
             await TestMissingAsync(@"
```

Comment:
I think the "Inline Local" was intentionally disabled for ref locals because the code action did not know how to do the inlining correctly in all cases and that caused crashes/asserts.
Until we sure that code actions can handle ref locals, we should keep the code action disabled.
Just disabling the test seems dangerous.